stats/lib/platform-stats-fetchers.js

```diff
@@ -1,16 +1,24 @@
 import assert from 'http-assert'
 import { today, yesterday } from './request-helpers.js'
 
-/** @typedef {import('@filecoin-station/spark-stats-db').Queryable} Queryable */
+/** 
+  @typedef {import('./typings.js').DateRangeFilter} DateRangeFilter 
+  @typedef {import('@filecoin-station/spark-stats-db').Queryable} Queryable 
+  @typedef {import('./typings.js').FastifyPg} FastifyPg 
+*/
+
+/**
+ * @param {FastifyPg} pg - Fastify pg object with database connections
+ */
 
 const ONE_DAY = 24 * 60 * 60 * 1000
 
 /**
- * @param {Queryable} pgPool
+ * @param {FastifyPg} pg 
  * @param {import('./typings.js').DateRangeFilter} filter
  */
-export const fetchDailyStationCount = async (pgPool, filter) => {
-  const { rows } = await pgPool.query(`
+export const fetchDailyStationCount = async (pg, filter) => {
```

Comment:
In case we pass correct database instance we don't need to change this handler.

Reply:
ping @Goddhi